### DISCUSSION TITLE
git没找到这样的功能，能开发一个吗

### DISCUSSION THREAD

**Discussion by noday** [pxzxj/VCSLabel#2] [Show and tell]
git没找到这样的功能，能开发一个吗

**Comment by pxzxj** [comment]
嗯 计划一两个月内开发好

**Reply by pxzxj** [reply]
额 你重启IDEA试试 我自己这儿是正常的

**Reply by pxzxj** [reply]
或者你把插件删掉重新装下 直接升级好像有时候有问题

**Reply by RainySecret** [reply]
不行啊，我卸载了，然后重启了idea，接着再搜索安装，然后也还重启了下idea，还是svn项目不显示，Git项目正常。
跟版本有关系么，我的idea版本是2020.3.4

**Comment by pxzxj** [comment]
已经开发完成，插件重命名成了VCSLabel，可以直接搜索安装
![image](https://user-images.githubusercontent.com/28888745/156276660-181f0ea5-143b-424d-bf72-8c9acf3748e2.png)
